app/graphql/resolvers/project_metadata_summary_resolver.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Resolvers
+  # Project Metadata Summary Resolver
+  class ProjectMetadataSummaryResolver < BaseResolver
+    type GraphQL::Types::JSON, null: false
+
+    argument :keys, [GraphQL::Types::String],
+             required: false,
+             description: 'Optional array of keys to limit metadata result to.',
```

Comment:
**nitpick (blocking):** Can you update this comment to say `metadata summary` instead of just `metadata`

Reply:
Fixed in https://github.com/phac-nml/irida-next/pull/751/commits/35d383e7fd87c42e347b93f46b78d18bc0bea5ec